### PR TITLE
Vcpu wait metric

### PR DESCRIPTION
**Pull request by xpivarc**
**What this PR does / why we need it**:

Exposing `kubevirt_vmi_vcpu_wait` metric. This can be used to identify workload suffering from poor I/O performance, ...
Bumping libvirt-go to 6.5.0 for this metric to be available in libvirt-go and aligning with our libvirt version(6.0.0).

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
Committed bump separately hopefully for a better review.

**Release note**:
```release-note
Vcpu wait metrics available
```

### CODE REVIEW

File: pkg/monitoring/vms/prometheus/prometheus.go [hunk 1]
```diff
@@ -95,6 +95,16 @@ var (
 		},
 		nil,
 	)
+
+	vcpuWaitDesc = prometheus.NewDesc(
+		"kubevirt_vmi_vcpu_wait",
+		"vcpu time spent by waiting on I/O",
```
[the remark this answers is not in the document]
Comment (xpivarc) [reply pkg/monitoring/vms/prometheus/prometheus.go:101]: You want to add it to name right?
While I am at this I checked in which units this is and other metrics. I see that vcpu is in seconds, libvirt expose it in microseconds and we divide by 1e+9(but it should be 1e+6). Am I missing anything, or we have a typo?

**Comment by danielBelenky** [comment]
@xpivarc can you please rebase this PR? I think that it's ready to be merged after that

File: go.sum [hunk 2]
```diff
@@ -713,8 +720,8 @@ kubevirt.io/containerized-data-importer v1.10.9 h1:wYCPyMbCLdn5tlZWW/lPYTMPtIOSf
 kubevirt.io/containerized-data-importer v1.10.9/go.mod h1:qF594BtRRkruyrqLwt3zbLCWdPIQNs1qWh4LR1cOzy0=
 kubevirt.io/qe-tools v0.1.6 h1:S6z9CATmgV2/z9CWetij++Rhu7l/Z4ObZqerLdNMo0Y=
 kubevirt.io/qe-tools v0.1.6/go.mod h1:PJyH/YXC4W0AmxfheDmXWMbLNsMSboVGXKpMAwfKzVE=
-libvirt.org/libvirt-go v5.0.0+incompatible h1:0kjba9uMMwHm036eU20ys53WFFn9KxcHC/ps9gEjlWA=
-libvirt.org/libvirt-go v5.0.0+incompatible/go.mod h1:CPoljLoiC2aEw+62g1rZXl2oXAJaNsrq4YCSmJOELek=
+libvirt.org/libvirt-go v6.5.0+incompatible h1:7hrVIm1onDKjIYpKyWqAtF6+ugYOVSnyaWnNpn+8tLI=
+libvirt.org/libvirt-go v6.5.0+incompatible/go.mod h1:CPoljLoiC2aEw+62g1rZXl2oXAJaNsrq4YCSmJOELek=
```
Comment (stu-gott) [inline go.sum:724]: Oops wait a moment. This is a big change! is it needed by this PR?

Comment (xpivarc) [reply go.sum:724]: @stu-gott  Yes afaik, I need a libvirt-go which includes my [patch.](https://gitlab.com/libvirt/libvirt-go/-/commit/a0815d68d2c703ca2e64d61ba2079db03fa4f4fd). It's tagged on 6.5.0.

**Pull request merged**